### PR TITLE
Improve questions

### PR DESCRIPTION
**Pull request by samuelim01**
## Description

Wrap delete questions into single network round trip and ensure questions table is responsive

## Checklist

- [x] I have updated documentation
- [x] All tests passing

## Screenshots (if applicable)

### CODE REVIEW

**Review by KhoonSun47 — CHANGES_REQUESTED** [review]
Looks good for the backend! Just need to do some minor changes for the README.

File: services/question/README.md [hunk 1]
```diff
@@ -422,3 +422,41 @@ curl -X DELETE http://localhost:8081/questions/21
...
+### Parameters:
+
+- `ids` (Required) - The IDs of the questions to delete.
+
+### Responses:
+
+| Response Code               | Explanation                                    |
+|-----------------------------|------------------------------------------------|
+| 200 (OK)                    | Success, the question is deleted successfully. |
+| 400 (Bad Request)           | The `id` was not specified                     |
```
[the remark this answers is not in the document]
Comment (samuelim01) [reply services/question/README.md:442]: aights 👍

File: services/question/README.md [hunk 2]
```diff
@@ -422,3 +422,41 @@ curl -X DELETE http://localhost:8081/questions/21
...
```
Comment (limcaaarl) [inline services/question/README.md:442]: I think this should be `ids` to match the param above.

**Review by limcaaarl — CHANGES_REQUESTED** [review]
Other than the minor typo, everything seems good and work fine 👍👍

File: services/question/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samuelim01) [reply services/question/README.md]: @KhoonSun47 I modified the parameters section above to be clearer, does that work?

[the remark this answers is not in the document]
Comment (KhoonSun47) [reply services/question/README.md]: Yeap, it works, thanks :)

**Review by KhoonSun47 — APPROVED** [review]
LGTM!

**Review by limcaaarl — APPROVED** [review]

**Pull request merged**